### PR TITLE
Ccl2 76 scheduling page

### PR DESCRIPTION
**Pull request by Yvv1214**
https://github.com/makeitMVPadmin/CoffeeChat/assets/39809411/e3d01e72-2695-4465-9028-11ea5e7c7f4d

i added the calendar link to the navbar to see upcoming and past bookings, the schedule page has been tweaked but i believe someone was redesigning the calendar? and the last page is green  so u can see the letters but i think its all set for the hi-fi
also it doesn't say Raj Dev because i was experimenting with how longer names would fit on the last part.

### CODE REVIEW

**Comment by Yvv1214** [comment]
with the conflicts fixed the i left scheduling page how someone else left it in hi-fi where they added the option to enter URL or a location though it will not take you to You are Booked page as their page uses a modal.

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -44,6 +45,22 @@ const Navbar = () => {
...
+          navData.isActive ? "navLinksActive" : "navLinks"
+        }
+      >
+        <div>
+          <Link to="/bookings">
+            <img className="navIcons" src={Calendar02} alt="Calendar Icon" />
+          </Link>
+        </div>
+        <Link to="/bookings" className="navLink">
+          Connect
```
Comment (Ant-Shell) [inline src/components/Navbar/Navbar.jsx]: Let's update this from `Connect` to `Calendar`

**Review by Ant-Shell — COMMENTED** [review]
Looks good overall, just needs one update on Line 60 of `Navbar.jsx`

**Review by Ant-Shell — APPROVED** [review]
Looks good, merging

**Pull request merged**